### PR TITLE
Geometric key xy, closes #445.

### LINKED ISSUE #445: Rename geometric_key2 to geometric_key_xy
Functions and methods for operations in 2D space are usually called name_xy to differentiate them from 3D counter parts.

I think [compas.utilities.maps.geometric_key2](https://github.com/compas-dev/compas/blob/9904ebc9cf1f798657885c8d62c99daa4593f923/src/compas/utilities/maps.py#L74) should be renamed for consistency.

I’ll open PR when the sun is up and I’m at my computer.

### PR DESCRIPTION
**Pull request by tetov**
### What type of change is this?

- [x] Name change of function

### Checklist

1. [x] Add the change to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading: `Added`, `Changed`, `Removed`.
1. [x] Run all tests on your computer (i.e. `invoke test`).
1. [x] If you add new functions/classes, check that:
   1. [x] Are available on a second-level import, e.g. `compas.datastructures.Mesh`.
   1. [ ] Add unit tests (especially important for algorithm implementations).

### Notes

See Also in docstrings should probably be styled differently in the sphinx template, wrong grey color and rounded corners. They could of course also be removed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- compas.utilities.maps.geometric_key2 renamed to geometric_key_xy
```
Comment (gonzalocasas) [inline CHANGELOG.md]: This goes under the `Changed` headline, not `Added`

File: src/compas/utilities/maps.py [hunk 2]
```diff
@@ -41,6 +41,10 @@ def geometric_key(xyz, precision=None, sanitize=True):
 
         # 3.142,3.142,3.142
 
+    See also
+    --------
+    geometric_key_xy: Create geometric keys for 2D coordinates
```
Comment (gonzalocasas) [inline src/compas/utilities/maps.py:46]: This will not create a link between the two, but you could, it's something like :meth:`geometry_key_xy` iirc

**Review by gonzalocasas — CHANGES_REQUESTED** [review]
Besides the tiny changelog fix request, it LGTM!

File: src/compas/utilities/maps.py (hunk 2 above)
Comment (tetov) [reply src/compas/utilities/maps.py:46]: See Also example

This works, and the link is correct..

File: CHANGELOG.md (hunk 1 above)
Comment (tetov) [reply CHANGELOG.md]: Sorry, fixed!

File: src/compas/utilities/maps.py (hunk 2 above)
Comment (gonzalocasas) [reply src/compas/utilities/maps.py:46]: OH! cool, didn't know! I thought it needed the whole directive thingy to link properly

**Review by gonzalocasas — APPROVED** [review]
LGTM

File: src/compas/utilities/maps.py (hunk 2 above)
Comment (tetov) [reply src/compas/utilities/maps.py:46]: [Thank Napoleon](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections)

:)

**Pull request merged**